rudder-web/src/main/resources/rudder-apache-common.conf

```diff
@@ -151,7 +151,6 @@ SetEnvIf X-API-Version "[^1]" api_deny
 # Link to Rudder documentation
 Alias /rudder-doc /usr/share/doc/rudder/html
 <Directory /usr/share/doc/rudder/html>
-  DirectoryIndex rudder-doc.html
```

Comment:
Removing this line causes it to revert to the default, which is index.html, which is what our docs now use.